docs/CNI/multus.md

```diff
@@ -17,6 +17,11 @@ kube_network_plugin_multus: true
 
 will install Multus and Calico and configure Multus to use Calico as the primary network plugin.
 
+Namespace isolation enables a mode where Multus only allows pods to access custom resources (the `NetworkAttachmentDefinitions`) within the namespace where that pod resides. To enable namespace isolation:
+```yml
+multus_namespace_isolation: true
+```
+
```

Comment:
Please check the `Kubespray CI - pre-commit` job log and fix it.

Reply:
Sorry but I get an error 500 when trying to access the logs.

Reply:
I guess that the linter is complaining about the missing blank line before the code block. It's fixed.